### PR TITLE
EE-14662: Remove duplicated database properties folder which was comm…

### PR DESCRIPTION
**Pull request by guysidebottom**
…ited by mistake on a previous ticket. Add database resource files so flyway drops the feedback table which is no longer required since the feedback form has been removed from IPS.

### CODE REVIEW

File: src/main/resources/db/migration/postgresql/V1_4__ALTER_TABLE_audit.sql [hunk 1]
```diff
@@ -1 +1 @@
-ALTER TABLE audit ALTER COLUMN detail type JSONB USING detail::JSONB;
+ALTER TABLE audit ALTER COLUMN detail type JSONB USING detail::JSONB;
```
Comment (mikehalmaho) [inline src/main/resources/db/migration/postgresql/V1_4__ALTER_TABLE_audit.sql]: Can you revert  this file - if it changes at all (even whitespace) then the entire update will be rejected by Flyway.

File: src/main/resources/db/migration/hsqldb/V1_5__DROP_TABLE_feedback.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS dbo.feedback;
```
Comment (Tim-Hammonds) [inline src/main/resources/db/migration/hsqldb/V1_5__DROP_TABLE_feedback.sql]: If all variants (hsqldb and postgres) use the same migration, you can just use the common one.

File: src/main/resources/db/migration/postgresql/V1_4__ALTER_TABLE_audit.sql (hunk 1 above)
Comment (guysidebottom) [reply src/main/resources/db/migration/postgresql/V1_4__ALTER_TABLE_audit.sql]: Whitespace reinstated on latest commit.

File: src/main/resources/db/migration/hsqldb/V1_5__DROP_TABLE_feedback.sql (hunk 2 above)
Comment (guysidebottom) [reply src/main/resources/db/migration/hsqldb/V1_5__DROP_TABLE_feedback.sql]: Resolved.

**Pull request merged**